packages/quick_actions/quick_actions/test/quick_actions_test.dart

```diff
@@ -2,90 +2,68 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:async';
-
-import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
 import 'package:quick_actions/quick_actions.dart';
+import 'package:quick_actions_platform_interface/platform_interface/quick_actions_platform.dart';
+import 'package:quick_actions_platform_interface/quick_actions_platform_interface.dart';
+import 'package:quick_actions_platform_interface/types/shortcut_item.dart';
 
 void main() {
-  TestWidgetsFlutterBinding.ensureInitialized();
+  WidgetsFlutterBinding.ensureInitialized();
```

Comment:
Is it necessary to ensure that the `WidgetsFlutterBinding` is initialised? Aren't we simply testing against a mock interface and therefore we don't really require the bindings to be initialised?